### PR TITLE
ETL modules trans strings, #8650

### PR DESCRIPTION
**Pull request by njkim**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Issues Solved

#8650, #8651

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

File: arches/app/templates/javascript.htm [hunk 1]
```diff
@@ -528,6 +528,41 @@
         grouping-error-title='{% trans "Settings Conflict: Remove this card from grouped card?" as groupingErrorTitle %} "{{ groupingErrorTitle|escapejs }}"'
         grouping-error-message='{% trans "The cardinality of this card cannot be changed until you remove it from being grouped with the ${cardName} card.  Do you want to remove this card from being grouped with the ${cardName} card" as groupingErrorMessage %} "{{ groupingErrorMessage|escapejs }}"'
         model-does-not-exist='{% trans "!! Referenced model does not exist -- Delete and select a new model !!" as modelDoesNotExist %} "{{ modelDoesNotExist|escapejs }}"'
+        upload-csv-zip-file='{% trans "Upload .csv or .zip File" as UploadCsvZipFile %} "{{ UploadCsvZipFile|escapejs }}"'
```
Comment (chrabyrd) [inline arches/app/templates/javascript.htm:531]: nit casing: `UploadCsvZipFile` -> `uploadCsvZipFile`

**Review by chrabyrd — CHANGES_REQUESTED** [review]
overall looks good just 1 thing

File: arches/app/templates/javascript.htm (hunk 1 above)
Comment (njkim) [reply arches/app/templates/javascript.htm:531]: Good catch, thanks.

**Comment by chrabyrd** [comment]
manually merged this branch

**Pull request closed without merging**